webapp/graphite/metrics/views.py

```diff
@@ -49,40 +50,77 @@ def index_json(request):
   return json_response_for(request, matches, jsonp=jsonp)
 
 
+def queryParamAsInt(queryParams, name, default):
+  if name not in queryParams:
+    return default
+
+  try:
+    return int(queryParams[name])
+  except Exception as e:
+    raise InputParameterError('Invalid int value {value} for param {name}: {err}'.format(
```

Comment:
This is using 2-space indent, instead of the conventional 4-space indent. Above, `handleInputParameterError()` mixes 4-space and 2-space indent.

(This is one of the things DeepSource is complaining about. I wondered for a minute, didn't the tests already run flake8? They do, but `setup.cfg` disables many flake8 warnings, including the indent one. I often like to disable some, but not that many...)

Reply:
Sorry if it sounds like I'm nit-picking not-really-relevant stuff, because it's easier than the actual work ... as a very casual very minor contributor, I wasn't aware of the style of most of the code base. It seems that indent is mixed in hundreds of places already.

It also seems to me that this project should gradually find palatable flake8 warnings to re-enable, long before it attempts to use something more sophisticated like DeepSource. I'll open a pull request that starts work on finding what flake8 rules can be re-enabled with the fewest line changes.

Reply:
@ploxiln thx, i definitely like the idea of standardizing the style, once we know what the standard is which we want to move to. I'm fine with pretty much any standard, important is just that we're using it consistently everywhere.